webhook/webhook.go

```diff
@@ -148,13 +149,16 @@ func (hook *WebHook) processAlerts() {
 
 			host, _ := a.Annotations[hook.config.ZabbixHostAnnotation]
```

Comment:
What about adding that here and just dropping all the additions below for more clarity?
```suggestion
			host, exists := a.Annotations[hook.config.ZabbixHostAnnotation]
			if !exists {
				host = hook.config.ZabbixHostDefault
			}
```

Reply:
Sounds good to me, I'm not super familiar with go but your idea looks cleaner